cmake/modules/AddSwift.cmake

```diff
@@ -109,17 +110,18 @@ function(_add_variant_c_compile_link_flags)
     ""
     ${ARGN})
 
-  set(result
-    ${${CFLAGS_RESULT_VAR_NAME}}
-    "-target" "${SWIFT_SDK_${CFLAGS_SDK}_ARCH_${CFLAGS_ARCH}_TRIPLE}")
+  # MSVC doesn't understand the option -target
+  if ("${CMAKE_C_COMPILER_ID}" STREQUAL "Clang")
```

Comment:
I didn't see that part of your review, but have addressed it and the other parts just now - sorry!